omegaconf/basecontainer.py

```diff
@@ -265,6 +265,39 @@ def convert(val: Node) -> Any:
 
         assert False
 
+    def _instantiate_structured_config_impl(conf, instance_data: Dict[str, Any]) -> Any:
```

Comment:
rename conf to self in the signature?

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/f16ad229015723c6e211d6e285e6566f0c698679.